.pre-commit-config.yaml

```diff
@@ -27,6 +27,14 @@ repos:
     hooks:
     -   id: pyupgrade
         args: [--py37-plus, --keep-runtime-typing]
+-   repo: https://github.com/pycqa/flake8
+    rev: 5.0.4
+    hooks:
+    -   id: flake8
+        args: ["--config=.flake8"]
+        additional_dependencies:
+          - "flake8-docstrings"
+          - "flake8-import-order"
```

Comment:
i don't think we need this, as we're already using an import order hook which you can see just above this one